src/main/java/uk/gov/hmcts/divorce/print/BulkPrintService.java

```diff
@@ -25,19 +30,28 @@
 
     private final SendLetterApi sendLetterApi;
     private final AuthTokenGenerator authTokenGenerator;
+    private final HttpServletRequest request;
+    private final DocumentManagementClient documentManagementClient;
+    private final IdamService idamService;
```

Comment:
normally we auto wired fields and also we don't have all args constructor on the service. So let's keep it consistent

Reply:
Added auto wired instead of all args  constructor.